### PR TITLE
ZENKO-3013 restore CI for 1.1 and 1.2

### PR DESCRIPTION
**Pull request by tcarmet**

### CODE REVIEW

File: eve/scripts/setup-k8s.sh [hunk 1]
```diff
@@ -1,10 +1,8 @@
 #!/bin/sh
 set -e
 
-echo "Replacing existing credentials"
-rm -rf /root/.kube
-mkdir -p /root/.kube
-echo "${CI_KUBECONFIG}" | envsubst > /root/.kube/config
+mkdir -p ~/.kube/
```
Comment (tcarmet) [inline eve/scripts/setup-k8s.sh]:
```suggestion
```

**Review by tcarmet — COMMENTED** [review]
reviewing myself

**Review by miniscruff — APPROVED** [review]

**Review by dora-korpar — APPROVED** [review]

**Pull request merged**